cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/dataset/input/BasicPartitionedFileSetInputContext.java

```diff
@@ -80,7 +127,7 @@ private PartitionKey getPartitionKey(URI inputPathURI) {
     }
     throw new IllegalArgumentException(
       String.format("Failed to derive PartitionKey from input path '%s' and path to key mapping '%s'.",
-                    inputPath, pathToPartitionMapping));
+                    inputPathURI, pathToPartitionMapping));
```

Comment:
What I want to avoid is if a rare condition here - as rare as it is, it may be common/systemic for some jobs or environment - that it can bring down the logging system. Can our log collection deal with log records of 30MB?